### DISCUSSION TITLE
"Sourcemap is likely to be incorrect"

### DISCUSSION THREAD

**Discussion by codylindley** [swordev/suid#184] [General]
Anyone else seeing this when doing a vite build?

"Sourcemap is likely to be incorrect: a plugin (suid) was used to transform files, but didn't generate a sourcemap for the transformation. Consult the plugin documentation for help"

I am using sourcemaps in the build i.e.

```
    build: {
        rollupOptions: {
            output: {
                assetFileNames: 'static/[name].[hash][extname]',
                chunkFileNames: 'static/[name].[hash].js',
                entryFileNames: 'static/[name].[hash].js',
            },
        },
        outDir: 'build',
        sourcemap: true,
        target: 'esnext',
    },
```

**Comment by juanrgm** [comment]
https://github.com/swordev/suid/commit/31310a8b8bc3ff4cd982fd81efb08612a50443c4
